phpipam/client/client.go

```diff
@@ -20,12 +22,20 @@ type Client struct {
 
 // NewClient creates a new client.
 func NewClient(s *session.Session) *Client {
+	log.SetHandler(logfmt.New(os.Stderr))
+
+	SetLevel(log.InfoLevel)
```

Comment:
But if you want to implement the logging levels, log level should be passed from the environment variable, if the environment variable does not correspond to one of the desired values, then set default `log.InfoLevel`

Reply:
Ok I will implement this.

Reply:
Tell if me if environment variable name is OK for you.
I don't know where to add some doc (at least environment variable reference).

Regards

Reply:
`PHPIPAMSDK_LOGLEVEL` - is good.
Can you please add information about loglevel to the [README.md](https://github.com/pavel-z1/phpipam-sdk-go/blob/master/README.md) file?

Reply:
Description added